slidedeck/src/slides/09-special-topics/02-react-next.pug

```diff
@@ -93,5 +63,10 @@ section
 
 section
   h3 Hooks
-  p Functional components will be gaining the ability to have State
-  p No real feature change, some people just prefer Functional Component style
+  p Can be thought of as reusable higher-order wrappers for Functional Components
+  p Mostly syntactic sugar - cleaner, simpler way to do things you can already do
+
+  ul
+    li Add State or lifecycle
+    li Connect to Context
+    li Create your own!
```

Comment:
We might want to create a few examples, not needed for this PR of course.

Reply:
I know that hooks exist, and that's where my knowledge ends. So imma leave that to you